app/javascript/packages/document-capture/context/file-base64-cache.ts

```diff
@@ -1,6 +1,6 @@
 import { createContext } from 'react';
 
-const FileBase64CacheContext = createContext(/** @type {WeakMap<Blob,string>} */ (new WeakMap()));
+const FileBase64CacheContext = createContext(new WeakMap());
```

Comment:
We feel like we may be missing something in the transition to TS, but `yarn run tsc` worked, `yarn test` had all files pass locally, and this is pretty similar to [i18n-context.ts](https://github.com/18F/identity-idp/blob/cf8764c2be19a834c3c7116d35bfccb44de41938/app/javascript/packages/react-i18n/i18n-context.ts).

Still we felt maybe there's a piece we're missing with the interface definition or method call output.

Reply:
One thing I'd suggest we do is provide the generic argument to `WeakMap` to preserve the type. Right now it's falling back to `any` type, and our project is configured to be pretty flexible with that.

```suggestion
const FileBase64CacheContext = createContext(new WeakMap<Blob, string>());
```